radis/misc/__init__.py

```diff
@@ -48,3 +48,9 @@
 from .progress_bar import ProgressBar
 from .signal import resample, resample_even
 from .utils import DatabankNotFound, NotInstalled, getProjectRoot
+
+# Checking `~radis.json` exist or not, if not then converting `~.radis` into `~/radis.json`
+try:
+    automatic_conversion()
+except:
+    raise ("Couldn't Convert `.radis` to `radis.json`")
```

Comment:
keep the inheritance of errors in the error log : 
```suggestion
except Exception as err:
    raise ValueError("Couldn't Convert `.radis` to `radis.json`") from err
```